the-example-app.swiftTests/DeepLinkTests.swift

```diff
@@ -1,109 +1,105 @@
-            coursesViewController.select(category: nil)
-            expectation.fulfill()
-        }
-        waitForExpectations(timeout: 10.0, handler: nil)
-    }
-
-    func testLessonRoute() {
-        UIApplication.shared.open(URL(string: "the-example-app-mobile://courses/hello-sdks/lessons/sdk-basics")!, options: [:], completionHandler: nil)
-
-        tester.waitForTappableView(withAccessibilityLabel: "SDK basics")
-
-        UIApplication.shared.open(URL(string: "the-example-app-mobile://courses/hello-sdks/lessons/example-app-summary")!, options: [:], completionHandler: nil)
-
-        tester.waitForTappableView(withAccessibilityLabel: "Summary")
-    }
-
-    func testSettingsRoute() {
-        UIApplication.shared.open(URL(string: "the-example-app-mobile://settings")!, options: [:], completionHandler: nil)
-
-        tester.waitForTappableView(withAccessibilityLabel: "English (United States)")
-
-        // Tapping will cause the system to scroll
-        tester.tapView(withAccessibilityLabel: "API: Preview")
-        tester.waitForTappableView(withAccessibilityLabel: "API: Preview")
-    }
-
-    func testInvalidRoute() {
-        UIApplication.shared.open(URL(string: "the-example-app-mobile://foo")!, options: [:], completionHandler: nil)
-
-        let expectedAccessibilityLabel = "Oops, something went wrong\nInvalid route \'foo\'"
-        tester.waitForTappableView(withAccessibilityLabel: expectedAccessibilityLabel)
-        tester.tapView(withAccessibilityLabel: "OK")
-        tester.waitForAbsenceOfView(withAccessibilityLabel: expectedAccessibilityLabel)
-    }
-
-    func testInvalidCredentialsRoutesToSettings() {
-        UIApplication.shared.open(URL(string: "the-example-app-mobile://courses?space_id=jnzexv31feqf")!, options: [:], completionHandler: nil)
-
-        let expectedAccessibilityLabel = """
-        • This field is required: Content Preview API access token
-        • This field is required: Content Delivery API access token
-
-        """
-        tester.waitForTappableView(withAccessibilityLabel: expectedAccessibilityLabel)
-    }
-
-    func testSpaceWithoutLessonCopyModulesStillRendersLesson() {
-        UIApplication.shared.open(URL(string: "the-example-app-mobile://courses/hello-sdks/lessons/fetch-draft-content?space_id=r3rkxrglg2d1&delivery_token=98b2548760939aff3910f23e0b97dc6376e6c7aec5ebf73c5f3424b36b721e50&preview_token=dc4d50c7d811f519d5037f92cbabc1312c822f674a066fa2bfcfc3077cbfb6b0&editorial_features=enabled&api=cpa")!, options: [:], completionHandler: nil)
-
-        tester.waitForView(withAccessibilityLabel: "Fetch draft content")
-
-        let expectedAccessibilityLabel = "New space detected"
-        tester.waitForTappableView(withAccessibilityLabel: expectedAccessibilityLabel)
-        tester.tapView(withAccessibilityLabel: "OK")
-        tester.waitForAbsenceOfView(withAccessibilityLabel: expectedAccessibilityLabel)
-    }
-}
+//
```

Comment:
Can this file be removed instead of commented?

Reply:
It will be added back in a matter of weeks, so I'd prefer to leave it.